DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/README.md

```diff
@@ -0,0 +1,362 @@
+| usv_from_eigens.hpp       | Device code                     | Contains kernel implementation of `USVFromEigens`.                                                                                                                                                   |
+#### Shared source file:
+| Shared file                                                                      | Type        | Content                                                                                           |
+|----------------------------------------------------------------------------------|-------------|---------------------------------------------------------------------------------------------------|
+| [streaming_covariance_matrix.hpp](../../include/streaming_covariance_matrix.cpp) | Device code | Contains kernel implementations of `StreamingCovarianceMatrix` that computes covariance matrix.   |
+| [streaming_eigen.hpp](../../include/streaming_eigen.hpp)                         | Device code | Contains kernel implementations of `StreamingEigen` that computes eigen values and eigen vectors. |
+| [streaming_qrd.hpp](../../include/streaming_qrd.hpp)                             | Device code | Contains kernel implementations of `StreamingQRD` that is used to compute orthogonalized matrix.  |
+
+### Input covariance matrix computation
+This design imports `streaming_covariance_matrix.hpp` from the shared [include](../../include) directory. The covariance computation in this design uses the `fpga_linalg::StreamingCovarianceMatrix` functor, with the `standardized` parameter of the `StreamingCovarianceMatrix` functor set to `false`.
+
+The covariance of input A is equal to the transpose of A multiplied by A:
+```math
+Cov(A) = A^{T}A
+```
+Therefore this kernel performs a matrix multiplication by blocks, as described in the [PCA](../pca/) reference design, which also uses this functor.
+
+### Eigenvalues and Eigenvectors computation
+This design imports `streaming_eigen.hpp` from the shared [include]() directory. The eigenvalues and eigenvectors of the input covariance matrix are computed using the `fpga_linalg::StreamingEigen` functor. These eigenvalues and eigenvectors are going to be used to construct the final outputs of SVD.
+
+This kernel works only with input of rank sufficient matrix (all columns are linearly independent). It produces an output `rank_deficent_flag` to indicate if the input matrix is not linearly independent. If the flag is set to 1, the result of the SVD is known to be incorrect.
+
+The [PCA](../pca/) reference design also uses this functor. A detailed explanation of the algorithm of computing eigenvalues can be found in [Eigen values and Eigen vectors computation](../pca/README.md#eigen-values-and-eigen-vectors-computation) in the PCA sample.
```

Comment:
```suggestion
The [PCA](../pca/) reference design also uses this functor. A detailed explanation of the algorithm of computing eigenvalues can be found in [Eigenvalues and Eigenvectors computation](../pca/README.md#eigen-values-and-eigen-vectors-computation) in the PCA sample.
```
Yohann needs to fix this spelling mistake in his code samples, but I won't put that on you in this PR.